Misc/NEWS.d/next/Library/2019-02-15-15-12-32.bpo-36003.D5t4_b.rst

```diff
@@ -0,0 +1,6 @@
+`socketserver.TCPServer.allow_reuse_address` default changed from ``False``
+to ``True`` on POSIX. `socketserver.TCPServer.request_queue_size` default
+changed from ``5`` to ``None`` in order to choose a default reasonable value
+(usually 128). `http.server.HTTPServer` and
```

Comment:
I think you have an exception here, check the build of Travis.

Reply:
fixed, thanks